localization/pose_instability_detector/CMakeLists.txt

```diff
@@ -22,6 +24,14 @@ if(BUILD_TESTING)
   )
 endif()
 
+install(
+  DIRECTORY src/
+  DESTINATION include/pose_instability_detector
+  FILES_MATCHING PATTERN "*.hpp"
+)
+
+ament_export_include_directories(include)
+
```

Comment:
If you want to be able to include this package from other modules, I think it is okay to explicitly create an include directory in the package.
https://autowarefoundation.github.io/autoware-documentation/main/contributing/coding-guidelines/ros-nodes/directory-structure/#include-and-src

Reply:
I followed the guidelines and revised the file structure. Thank you!
https://github.com/autowarefoundation/autoware.universe/pull/7042/commits/a462cf3340a311260f69a8cb954a45367b31a4e4 https://github.com/autowarefoundation/autoware.universe/pull/7042/commits/ff5218c735a3834cd5d0d575dc24f50199e43326

Reply:
Oh, I remember we need to create an autoware directory. It is a good.